src/modules/PackageSlideShow.rb

```diff
@@ -1008,51 +842,54 @@ def DisplayGlobalProgress
       nil
     end
 
+    # Callback when file is downloaded ( but not yet installed )
+    # @param error[Integer] error code
     def DoneProvide(error, _reason, _name)
-      if error.zero?
-        @total_downloaded = Ops.add(@total_downloaded, @current_provide_size)
-
-        @total_count_downloaded = Ops.add(@total_count_downloaded, 1)
-        Builtins.y2milestone(
-          "Downloaded %1/%2 packages",
-          @total_count_downloaded,
-          @total_count_to_download
-        )
+      return if error.nonzero?
 
-        # move the progress also for downloaded files
-        UpdateTotalProgressValue()
-
-        d_mode = Ops.get_symbol(Pkg.CommitPolicy, "download_mode", :default)
-
-        if d_mode == :download_in_advance ||
-            d_mode == :default && Mode.normal &&
-                !Installation.dirinstall_installing_into_dir
-          # display download progress in DownloadInAdvance mode
-          # translations: progress message (part1)
-          SlideShow.SetGlobalProgressLabel(
-            Ops.add(
-              _("Downloading Packages..."),
-              # progress message (part2)
-              Builtins.sformat(
-                _(" (Downloaded %1 of %2 packages)"),
-                @total_count_downloaded,
-                @total_count_to_download
-              )
+      @total_downloaded += @current_provide_size
+
+      @total_count_downloaded += 1
+      Builtins.y2milestone(
+        "Downloaded %1/%2 packages",
+        @total_count_downloaded,
+        @total_count_to_download
+      )
+
+      # move the progress also for downloaded files
+      UpdateTotalProgressValue()
+
+      d_mode = Ops.get_symbol(Pkg.CommitPolicy, "download_mode", :default)
+
+      if d_mode == :download_in_advance ||
+          d_mode == :default && Mode.normal &&
+              !Installation.dirinstall_installing_into_dir
+        # display download progress in DownloadInAdvance mode
+        # translations: progress message (part1)
+        SlideShow.SetGlobalProgressLabel(
+          Ops.add(
+            _("Downloading Packages..."),
+            # progress message (part2)
+            Builtins.sformat(
```

Comment:
I think we have "format" for that now.

Reply:
well, here I keep it intentionally as I do not want to break translation.

Reply:
Well it is for master. So I am assuming changes anyway.